### PR TITLE
add files for landingpage

### PR DESCRIPTION
**Pull request by rkweehinzmann**
initial step for adding lp as service to scicatlive

### CODE REVIEW

File: services/landingpage/compose.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+services:
+  landingpage:
+
+    image: ghcr.io/scicatproject/landingpageserver:4a9653494e3fd31598ab46df5482e9e904f3b212
+
+    ports:
```
Comment (minottic) [inline services/landingpage/compose.yaml:6]: I think everything from line 6 onwards can be removed

**Review by minottic — COMMENTED** [review]
the yaml linting is broken

**Comment by minottic** [comment]
let's try to fix here, it's going to be confusing to have new PRs just for fixes. Just do new commits

**Comment by rkweehinzmann** [comment]
this commit https://github.com/SciCatProject/scicatlive/pull/185/commits/396f9ec8eeed586f27267e3dcbd10cf7a88e96de should fix the errors

**Comment by minottic** [comment]
the tests have just rerun by they brake. An easy fix is to use the image from latest for the LP and use the usual image:latest format. anyway we will need to change that later.

File: services/landingpage/compose.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+services:
+  landingpage:
+    image: ghcr.io/scicatproject/landingpageserver:latest
+
+    ports:
+      - 4201:80
+
+    user: 0:0
```
Comment (minottic) [inline services/landingpage/compose.yaml:8]: could we add this block: 

```yaml
profiles: 
 - search
```
as [here](https://github.com/SciCatProject/scicatlive/blob/main/services/searchapi/compose.yaml)

**Review by minottic — COMMENTED** [review]
now that it runs the tests, can we please tackle the comments? Also the home page readme will need to be updated, both in the diagram and in the configuration section

File: README.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ They are used when adding new services or grouping services together (and do not
...
 |---------|--------------------|---------------------------------------------------------------------------------|---------|-----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------|
-| profile | `COMPOSE_PROFILES` | <li>`analysis`: jupyter<li>`search`: searchapi<li>`'*'`: jupyter,searchapi</li> | `''`    | *                     | <li>analysis: enables additional jupyter notebook with python SciCat SDK installed and example notebooks<li>search: enables a SciCat interface for standardised search |                         |
+| profile | `COMPOSE_PROFILES` | <li>`analysis`: jupyter<li>`search`: searchapi, landingpage<li>`'*'`: jupyter,searchapi</li> | `''`    | *                     | <li>analysis: enables additional jupyter notebook with python SciCat SDK installed and example notebooks<li>search: enables a SciCat interface for standardised search |                         |
```
Comment (minottic) [inline README.md:40]: could you please add `and a public interface for published datasets`

`<li>search: enables a SciCat interface for standardised search`

**Comment by minottic** [comment]
blocked by #183

File: services/landingpage/README.md
Comment (minottic) [inline services/landingpage/README.md:1]: this will need to be reviewed after #183 is closed

File: services/landingpage/compose.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+services:
+  landingpage:
+    image: ghcr.io/scicatproject/landingpageserver:latest
```
Comment (minottic) [inline services/landingpage/compose.yaml:3]: the tag will need to be reviewed after https://github.com/SciCatProject/scicatlive/issues/183 is closed. Fine to keep it as is for now

also `config` might be added, together with the folder and the related files

**Comment by minottic** [comment]
depends on: https://github.com/SciCatProject/LandingPageServer/pull/375

**Pull request closed without merging**